Doc/library/subprocess.rst

```diff
@@ -654,6 +655,16 @@ functions.
    .. versionadded:: 3.10
       The ``pipesize`` parameter was added.
 
+   *read_cb* can be used to replace the default handler for reading the
```

Comment:
"to replace the default _internal_ handler for"

and strike the ", and is subprocess.Popen._read_handler" text.  we don't need to tell people what the internal-only thing is in the public API docs.  just that they can replace it.

Describe the required callable signature in the docs here and how it is expected to behave.  Don't force people to read subprocess.py in order to successfully implement a read_handler.